src/callbacks_step/analysis.jl

```diff
@@ -171,6 +172,7 @@ function initialize!(cb::DiscreteCallback{Condition,Affect!}, u_ode, t, integrat
   end
 
   analysis_callback.start_time = time_ns()
+  analysis_callback.start_gc_time = Base.gc_time_ns()
```

Comment:
I asked on Slack whether we can get this to stable. If not, my suggestion would be to add a corresponding comment that this might break and how to fix it (i.e., what to remove 😬). And yes, I will pledge to fix it if it breaks in later versions :-)